pyvista/plotting/plotting.py

```diff
@@ -2403,7 +2403,17 @@ def add_volume(self, volume, scalars=None, clim=None, resolution=None,
         -------
         vtk.vtkActor
             VTK actor of the volume.
-
+        
+        Examples
+        --------
+        >>> from pyvista import examples
+        >>> bolt_nut = examples.download_bolt_nut()
+        >>> 
+        >>> import pyvista as pv
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_volume(bolt_nut, cmap="coolwarm")
+        >>> pl.show()
```

Comment:
This looks good, thanks for adding it! Just some small style nitpicking:

We usually have imports first, and no empty lines in the examples. Just pointing this out for the sake of consistency.

And please consider adding a short description for the example. See e.g. https://github.com/pyvista/pyvista/blob/8a807e8a77025c649d8dd8339a31763d9f6bd05d/pyvista/core/filters/poly_data.py#L437-L450

Reply:
Thank you for the quick review. I have just revised the example as you suggested.